network/kademlia_test.go

```diff
@@ -370,80 +390,110 @@ func binStr(a *BzzAddr) string {
 	return pot.ToBin(a.Address())[:8]
 }
 
-func TestSuggestPeerFindPeers(t *testing.T) {
-	tk := newTestKademlia(t, "00000000")
-	tk.On("00100000")
-	tk.checkSuggestPeer("<nil>", 0, false)
+//Tests peer suggestion. Depends on expectedMinBinSize implementation
+func TestSuggestPeersFindPeers(t *testing.T) {
```

Comment:
Please consider renaming this function. It sounds a bit nonsensical to me.

Reply:
"TestSugestPeers" ?

Reply:
Sounds like a good idea.